acceptance_tests/features/deactivate_uac.feature

```diff
@@ -0,0 +1,16 @@
+
+Feature: deactivate UACs for a case
+
+  Scenario: A case is loaded, action rule is triggered and UAC is deactivated for that case
+    Given sample file "sample_1_limited_address_fields.csv" is loaded successfully
+    And a print action rule has been created with template "["__uac__"]" and classifiers "1=1"
```

Comment:
Shouldn't need the workaround `"1=1"` classifiers here or anywhere in the AT's now it's allowed to be empty

Reply:
Perhaps instead they should have the "default" classifiers which are no longer applied automatically.

Reply:
Needs discussion... behave doesn't allow empty values

Reply:
Changed to have its own step without classifiers as @AdamHawtin recommended